src/components/Home/HardwareCard/HardwareCard.tsx

```diff
@@ -0,0 +1,90 @@
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+import { HardwareInfo } from "../../../models/hardware-info";
+import LoadingBox from "../../Shared/LoadingBox/LoadingBox";
+
+const PI_NUM_CORES = 4;
+
+export type Props = {
+  hardwareInfo: HardwareInfo | null;
+};
+
+export const HardwareCard: FC<Props> = ({ hardwareInfo }) => {
+  const { t } = useTranslation();
+
+  if (!hardwareInfo) {
+    return (
+      <div className="h-full w-1/2">
+        <LoadingBox />
+      </div>
+    );
+  }
+
+  const {
+    cpu_overall_percent,
+    temperatures_celsius,
+    disks,
+    vram_usage_percent,
+  } = hardwareInfo;
+
+  const system_temp = temperatures_celsius.system_temp;
```

Comment:
Do we internally want to stick to lowerCamelCase?

Reply:
My bad, sorry :+1: 
Thank you!